### ISSUE TITLE
Unable to create email template: Passed form must inherit TranslationFormInterface

### ISSUE THREAD

**Issue by vtsykun** [oroinc/platform#900]
**STR**
1) Install application with `es_ES` localization (oro/platform 3.1.4) 

You can take configuration from https://github.com/oroinc/orocommerce-application-de/blob/master/config/config.yml
```yml
parameters:
    locale: es_ES

#config.yml
....
framework:
    translator:
        paths:
            - '%kernel.project_dir%/translations'
        fallback: en
    default_locale:           "en"
.....
oro_locale:
    formatting_code: '%locale%'
    language: '%locale%'
```

2) Open email template create page
http://localhost/email/emailtemplate/create

![Selection_999(170)](https://user-images.githubusercontent.com/21358010/54382459-16efbd00-46a1-11e9-8680-cf3be7f3300e.png)

**Comment by mbessolov** [comment]
_Internal ticket ID: BAP-18560_

**Comment by anyt** [comment]
Fixed in 3.1.6.
Thank you for the contribution!
